src/openvic-simulation/military/UnitType.hpp

```diff
@@ -94,7 +94,14 @@ namespace OpenVic {
 	struct UnitTypeBranched<UnitType::branch_t::LAND> : UnitType {
 		friend struct UnitTypeManager;
 
-		enum struct allowed_cultures_t { ALL_CULTURES, ACCEPTED_CULTURES, PRIMARY_CULTURE };
+		// Each value is a subset of its predecessor, so smaller values contain larger values
+		enum struct allowed_cultures_t { ALL_CULTURES, ACCEPTED_CULTURES, PRIMARY_CULTURE, NO_CULTURES };
```

Comment:
Why add NO_CULTURES?
That value is invalid and never occurs in Victoria 2 (including mods).

Reply:
Nevermind, figured it out. You use this purely as a default in the country instance so you can expand it when unlocking more types.